metals/src/main/scala/scala/meta/internal/builds/ShellRunner.scala

```diff
@@ -130,3 +133,80 @@ class ShellRunner(
   }
 
 }
+
+object ShellRunner {
+
+  /**
+   * Note: if returning a [[Future]] is more suitable, all you
+   * need to do is to uncomment the commented code.
+   */
+  def quickRunWithShortStringResult(
```

Comment:
Why not add it along the other methods? We could cache the Java home in that case inside the class.

```suggestion
  def runSync(
```

Reply:
Do you access it in the tests? I can't find it.

Reply:
tests/unit/src/test/scala/tests/troubleshoot/ProblemResolverSuite.scala is calling this static method:

`JdkVersion.maybeJdkVersionFromJavaHome(javaHome.map(AbsolutePath(_)))`

which in turn is calling `quickRunWithShortStringResult` as a static method.

Reply:
Ok, I guess we could leave it as is, but please try and cache the value maybe in an atomic inside the object?.